renovate.json

```
@@ -6,10 +6,24 @@
     "before 5am on Monday"
   ],
   "packageRules": [
+    {
+      "matchPackagePatterns": ["*"],
+      "matchUpdateTypes": ["major"],
+      "groupName": "all major dependency bump",
+      "groupSlug": "all-major",
+      "automerge": true
+    },
+    {
+      "matchPackagePatterns": ["*"],
+      "matchUpdateTypes": ["minor"],
+      "groupName": "all minor dependency bump",
+      "groupSlug": "all-minor",
+      "automerge": true
+    },
```

We don't want to automerge every major/minor release. Btw. do you remember how the order of packageRules is read by renovate bot? I'm wondering if these two sections shouldn't be at the bottom of this array.
```suggestion
    {
      "matchPackagePatterns": ["*"],
      "matchUpdateTypes": ["major"],
      "groupName": "all major dependency bump",
      "groupSlug": "all-major"
    },
    {
      "matchPackagePatterns": ["*"],
      "matchUpdateTypes": ["minor"],
      "groupName": "all minor dependency bump",
      "groupSlug": "all-minor"
    },
```

---

right, thanks I had this but redo and forget :(